content/en/getting_started/code_analysis/_index.md

```diff
@@ -302,6 +308,72 @@ Click on a service to access information about CI pipelines from Pipeline Visibi
 
 {{< img src="/getting_started/code_analysis/catalog_service.png" alt="A link to the source code directly in GitHub from a detected library vulnerability" style="width:100%" >}}
 
+### Linking services to code violations and libraries
+
+Datadog associates code violations or libraries with relevant services by using the following mechanisms:
+
+1. [Identifying the code location associated with a service using the Service Catalog.](#identifying-the-code-location-in-the-service-catalog)
+2. [Detecting usage patterns of files within additional Datadog products.](#detecting-file-usage-patterns)
+3. [Searching for the service name in the file path or repository.](#detecting-service-name-in-paths-and-repository-names)
+
+If one method succeeds, no further matching attempts are made. Each mapping method is detailed below.
```

Comment:
```suggestion
If one method succeeds, no further matching attempts are made. Each mapping method is detailed below.
```
Is this supposed to say "each matching method" ?

Reply:
In this instance, we're talking about the mapping method, which is the process of associating and linking code violations and libraries with services. With "matching method", that could imply a focus on finding exact matches only, and we have multiple options in case one of them doesn't work out. Updating the first one to be more clear.